code/cynthia/Python/test_ATM_lab12.py

```diff
@@ -0,0 +1,37 @@
+#Automated Tests
+#using using ATM 
+
+from lab08_2 import ATM
+import pytest
+from unittest.mock import patch 
+
+def test_ATM_creation():
+    atm = ATM(0)
+    assert atm.balance == 0
+    atm = ATM(-5)
+    assert atm.balance == -5
+    atm = ATM(.1)
+    assert atm.interest_rate == .1
+
+def test_check_balance():
+    atm = ATM(0)
+    assert atm.check_balance() == 0
```

Comment:
Add some more assertions here! Each test should have _at least_ two assertions, even if they're very similar.